tests/backends/tests.py

```diff
@@ -128,6 +128,19 @@ def test_order_of_nls_parameters(self):
             cursor.execute(query)
             self.assertEqual(cursor.fetchone()[0], 1)
 
+    def test_sequence_name_truncation(self):
+        """Sequence names are truncated as expected."""
+        seq_name = connection.ops._get_no_autofield_sequence_name('schema_authorwithevenlongee869')
```

Comment:
`_get_no_autofield_sequence_name` doesn't exist in `stable/1.11.x` branch. Please use `_get_sequence_name` instead.